migration.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+
+$projects_dir = "_data/projects/"
+
+def add_project p
+  # flatten the node
+  contents = {
+    :name => p["name"],
+    :desc => p["desc"],
+    :site => p["site"],
+    :tags => p["tags"],
+    :upforgrabs => p["upforgrabs"]
+  }
+
+  # dump the file to disk in the new location
+  new_filename = p["name"].downcase
```

Comment:
I've codified the rules here for mapping existing project names to this new structure. Thoughts?